litho-core/src/main/java/com/facebook/litho/InternalNode.java

```diff
@@ -205,6 +209,8 @@ void init(YogaNode yogaNode, ComponentContext componentContext, Resources resour
     mResourceResolver.init(
         mComponentContext,
         componentContext.getResourceCache());
+
+    mIsBuilt = false;
```

Comment:
My assumption here was that InternalNode objects are pooled in ComponentPool, so wouldn't we need to reset  the everytime init() is called?